samples/NativeLibrary/README.md

```diff
@@ -28,17 +28,67 @@ The above command will drop a static library (Windows `.lib`, OSX/Linux `.a`) in
 
 The above command will drop a shared library (Windows `.dll`, OSX `.dylib`, Linux `.so`) in `./bin/[configuration]/netstandard2.0/[RID]/publish/` folder and will have the same name as the folder in which your source file is present.
 
+### Loading shared libraries from C and importing methods
+
+For reference, you can read the file LoadLibrary.c.
+The first thing you'll have to do in order to have a proper "loader" that loads your shared library is to add these directives
+
+```c
+#ifdef _WIN32
+#include "windows.h"
+#define symLoad GetProcAddress GetProcAddress
+#define libClose FreeLibrary
+#else
+#include "dlfcn.h"
+#define symLoad dlsym
+#define libClose dlclose
+#endif
+```
+
+After these, in order to load the 'handle' of the shared library
+
+```c
+#ifdef _WIN32
+HINSTANCE handle = LoadLibrary(path);
+#else
+void *handle = dlopen(path, RTLD_LAZY);
+#endif
+  ```
```

Comment:
There are extra two spaces at the start of the line. Is suspect that it may be the reason for the odd rendering for the rest of the file.